packages/api/lib/granules.js

```diff
@@ -311,11 +311,28 @@ async function getGranulesForPayload(payload) {
   return uniqueGranules;
 }
 
+async function getUniquePgGranuleByGranuleId(knex, granulePgModel, granuleId) {
```

Comment:
I'm wondering if this is a candidate for the db package?

Reply:
Yup! I think it's much better there.

https://github.com/nasa/cumulus/pull/2415/commits/76ca4a1ba02d51c2753f87b8c88f48d430e3219f